shared/lib/wpcom-undocumented/lib/me.js

```diff
@@ -1,6 +1,8 @@
 /**
  * Module dependencies.
  */
+import _Me_ from 'wpcom-unpublished/dist/lib/me';
```

Comment:
Just trying to note the difference between this Class and the parent Class. I've brought this notation from wpcom.js. Changing ...